handler.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+const uuid = require('uuid');
+const AWS = require('aws-sdk'); 
+AWS.config.setPromisesDependency(require('bluebird'));
+const dynamoDb = new AWS.DynamoDB.DocumentClient();
+
+/* POST /guests */
+module.exports.submitGuest = async (event) => {
+	const requestBody = JSON.parse(event.body);
+  const firstName = requestBody.firstName;
+  const lastName = requestBody.lastName;
+  const responded = requestBody.responded;
+
+	if (typeof firstName !== 'string' || typeof lastName !== 'string' || typeof responded !== 'boolean') {
+    return {
+      statusCode: 400,
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Methods': 'OPTIONS, POST, GET',
+      },
+      body: JSON.stringify({
+        message: `Validation failed. Invalid input type(s).`,
```

Comment:
Did you mean for this to be a template literal using backticks ``` ` ```?
```suggestion
        message: 'Validation failed. Invalid input type(s).',
```